File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 1 - C/q3.c
//Faça um algoritmo que receba um salário e um percentual de aumento e depois e imprima o novo valor do salário. 
#include<stdio.h>
int main(){
	float salario, percentual_aument, novo_salario;
	printf("Insira o salario: ");
	scanf("%f", &salario);
	printf("Insira o percentual de aumento: ");
	scanf("%f", &percentual_aument);
	novo_salario=salario+(salario*percentual_aument)/100;
	printf("Salario novo: %f\n", novo_salario);	
	system("pause");
}

File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 1 - Python/q3.py
#Faça um algoritmo que receba um salário e um percentual de aumento e
#depois e imprima o novo valor do salário.

salario=float(input("Insira o salario do funcionario: "))
percentual_aument=float(input("Insira o percentual de aumento: "))
print("Aumento de ", percentual_aument,"%")
novo_salario=salario+(salario*percentual_aument)/100
print("Salario novo: ", novo_salario)
